docs/subset15.md

```diff
@@ -0,0 +1,408 @@
+# Language subset 15
```

Comment:
We should add this after we tag subset 14

Reply:
Fixed